Resources/Locale/en-US/game-ticking/game-ticker.ftl

```diff
@@ -33,7 +33,8 @@ player-first-join-message = Player {$name} joined for the first time.
 # Displayed in chat to admins when a player leaves
 player-leave-message = Player {$name} left.
 
-latejoin-arrival-announcement = {$character} ({$job}) has arrived at the station!
+latejoin-arrival-announcement = {$character} ({$job}) { CONJUGATE-HAVE($entity) } arrived at the station!
+latejoin-arrival-announcement-special = {$character}, The {$job}, { CONJUGATE-HAVE($entity) } stepped aboard the station!
```

Comment:
Just a small thought, but I think, "on deck!" fits better, and perhaps you could even rearrange things to be, "Captain [name], on deck!"

Reply:
{$job} {$character} { CONJUGATE-HAVE($entity) } on deck!

Reply:
To me, "on deck" implies that the captain has steped onto the station which might not be the case (eg. they are on the arrivals terminal).